assets/enx-redirect/report/index.html

```diff
@@ -0,0 +1,108 @@
+{{define "report/index"}}
+
+{{$currentRealm := .realm}}
+
+<!doctype html>
+<html dir="{{$.textDirection}}" lang="{{$.textLanguage}}">
+<head>
+  {{template "report/head" .}}
+</head>
+
+<body class="tab-content">
+  <main role="main" class="container">
+    <h1>Request exposure notifications verification code from {{.realm.Name}} ({{.realm.RegionCode}})</h1>
+
+    <form method="POST" action="/report/issue" class="floating-form">
+      {{ .csrfField }}
+
+      <div id="form-area">
+
+        <div class="card mb-3 shadow-sm">
```

Comment:
I think, for self-report, we need to reduce this to a single field. People are going to get confused if we don't. Additionally, "symptom onset" is a very technical word.

Reply:
I think that's fine for now - this is going through UX design

Reply:
meaning - I removed it for now, but it may come back